atat_provisioning_api.yaml

```diff
@@ -259,6 +259,4 @@ components:
           type: integer
         content:
           type: object
-          additionalProperties: true
-        request:
-          $ref: '#/components/schemas/CspInvocation'
\ No newline at end of file
+          additionalProperties: true
```

Comment:
I see that `CspResponse` is now:
```
CspResponse:
      required:
        - code
        - content
        - request
      properties:
        code:
          type: integer
        content:
          type: object
          additionalProperties: true
```
`request` is required, but it is not defined in properties?
I see the `$ref` was deleted for it, but shouldn't we at least define it as an object in the yaml?